### PR TITLE
Set autoheight on dashboard tables

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Review by mamy-CS — APPROVED** [review]
Oh nice, I didn't see the autoHeight props there. LGTM.

File: tornjak-frontend/src/components/dashboard/table/dashboard-table.js [hunk 1]
```diff
@@ -16,13 +16,7 @@ class TableDashboard extends React.Component {
...
 
   render() {
-    const { numRows, tableType, data, columns, title } = this.props, viewConst = 70, limitedViewRows = 5;
-    var tableHeight = 0;
-    if (tableType === "expandedView") {
-      if (data.length < 2) {
-        tableHeight = 200; //const table height if numRows is less than 2
-      } else { tableHeight = data.length * viewConst; } //multiply by a constant to keep table height consistent}
-    } else if (tableType === "limitedView") { tableHeight = limitedViewRows * viewConst; }
+    const { numRows, tableType, data, columns, title } = this.props;
```
Comment (mamy-CS) [inline tornjak-frontend/src/components/dashboard/table/dashboard-table.js:19]: tableType, not used here. Remove.

**Review by mamy-CS — CHANGES_REQUESTED** [review]

**Comment by lumjjb** [comment]
@mamy-CS removed the tableTypes, ready for review again.

**Review by mamy-CS — APPROVED** [review]
lgtm!

**Review by maia-iyer — APPROVED** [review]
lgtm!

**Pull request merged**